src/components/AdminPortal/EventRegistrantsModal/Modal/EventRegistrantsModal.spec.tsx

```
@@ -106,6 +110,98 @@ vi.mock('components/NotificationToast/NotificationToast', () => ({
+              key={option.id}
+              data-testid={`option-${option.id}`}
+              onClick={(): void => {
+                if (onChange) {
+                  onChange({} as React.SyntheticEvent, option);
+                }
+              }}
+              onKeyDown={(): void => {
+                /* mock */
+              }}
+              role="button"
+              tabIndex={0}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)